JSDemos/Demos/DataGrid/BatchUpdateRequest/description.md

```diff
@@ -0,0 +1,15 @@
+If **editing**.[mode](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/#mode) is *"batch"*, end-users can modify multiple records and submit all changes simultaneously. If data is stored on a server, the **DataGrid** component sends multiple requests to save edited objects - one request per object. This is because most servers process only one edit operation at a time. However, if your server supports batch update, you can configure the **DataGrid** to save all changes with a single request.
+
+To add this functionality, implement the [onSaving](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onSaving) function. It accepts an `e` object that contains fields necessary for batch update. The following steps describe the function's implementation:
```

Comment:
"fields necessary for batch update" means what? Why are they necessary?

Reply:
The `e` object contains fields that can be used to configure batch update. For example, e.cancel, e.changes.

Reply:
```suggestion
To add this functionality, implement the [onSaving](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onSaving) function. It accepts an `e` object that contains fields used for batch update. The following steps describe the function's implementation:
```